.github/CONTRIBUTING.MD

```diff
@@ -15,6 +15,10 @@ To explore our components and their documentation we use storybook. To start the
 You're ready to add a new component to this library, THANK YOU! However, this is only a [PRESENTATIONAL](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) component library. Please do not add components with any dependencies on internationalization libraries, redux, or react-router.
 Feel free to add any other types of lower-level dependencies you might need for your components (i.e. lodash, etc), but please use `yarn add` to add dependencies to the project not `npm install`.
```

Comment:
How do you go about getting IBM OS approval for these libraries?  It seems scary to tell devs to just include whatever libraries they need/want without talking about the process.  I would have expected to see something almost opposite, where, DO NOT add new libraries to the project without first providing a discussion about why it's required and has it been approved.

Reply:
We should update this to include context around how including a new entry in `dependencies` requires internal products to re-certify their open source licensing approvals, which is not always a trivial task. 

`devDependencies` are less of a concern as they're not included in the final export.

In either case, justification should accompany the PR including changes to dependencies.